Client/Frontend/Strings.swift

```diff
@@ -3897,10 +3897,10 @@ extension String {
             value: "Learn more",
             comment: "Label for the Learn more button in the Shopping Experience Opt In onboarding Card (Fakespot)")
         public static let OptInCardDisclaimerText = MZLocalizedString(
-            key: "Shopping.OptInCard.Disclaimer.Title.v120",
+            key: "Shopping.OptInCard.Disclaimer.Text.v120",
```

Comment:
That means we should backport this PR to 120 as well 🤔 ? Cause that string will be deleted in the 120 release otherwise if we don't. Unless we don't care since this string won't be accessible to public yet?

Reply:
Yes, this needs to be backported. But as Fakespot will only be released for en-US this change shouldn't have an impact on the user experience because of missing translations.

Reply:
I am checking because I'm not sure about the latest statement. For example, I'm in the USA but I use Firefox for Android in French. These strings have all been translated in French. So I would assume I would get the experiment translated in French. Can we have someone verify this? Will also ask in Fakespot mobile channel

Reply:
Confirmed on Slack that this is just targeted to en-US users. So this should be good to go. Thanks for your patience :)